apps/desktop/desktop/src/main/java/bisq/desktop/main/content/support/resources/ResourcesView.java

```diff
@@ -74,10 +74,15 @@ public ResourcesView(ResourcesModel model, ResourcesController controller) {
 
         Label localVersionHeadline = SettingsViewUtils.getHeadline(Res.get("support.resources.localVersion.headline"));
 
+        String versionString = ApplicationVersion.getVersion() != null ?
+            ApplicationVersion.getVersion().getVersionAsString() : "Unknown";
```

Comment:
For undefined data we usually use `Res.get("na)` which is hte i18n string for `N/A` (Not Available).